packages/gestalt/src/Datapoint/InternalDatapoint.js

```diff
@@ -0,0 +1,117 @@
+// @flow strict
+import { type Node } from 'react';
+import Badge from '../Badge.js';
+import Flex from '../Flex.js';
+import Icon from '../Icon.js';
+import TapArea from '../TapArea.js';
+import Text from '../Text.js';
+import Tooltip from '../Tooltip.js';
+import { type Indexable } from '../zIndex.js';
+import DatapointTrend from './Trend.js';
+
+type BadgeObject = {|
+  text: string,
+  type?: 'info' | 'error' | 'warning' | 'success' | 'neutral' | 'darkWash' | 'lightWash',
+|};
+
+type TrendObject = {|
+  accessibilityLabel: string,
+  value: number,
+|};
+
+type Props = {|
+  /**
+   * Adds a badge to the title. Currently a beta feature, expect changes.
```

Comment:
ditto here, no need for these comments now that this is internal. Only worth calling out what's different from Datapoint